src/tooltip.js

```diff
@@ -157,6 +158,7 @@ class Tooltip extends Component {
 
   componentWillUnmount() {
     Dimensions.removeEventListener('change', this.updateWindowDims);
+    this.interactionPromise && this.interactionPromise.cancel();
```

Comment:
small nit-pick, but for readability will you use this like you did below here, please?

```
if (this.interactionPromise) {
     this.interactionPromise.cancel();
}
```